appveyor.yml

```diff
@@ -0,0 +1,23 @@
+build: false
+shallow_clone: false
+platform: 'x86'
+clone_folder: c:\projects\symfony
+
+environment:
+  global:
+    PHP: c:/PHP
+
+init:
+  - SET PATH=c:\php\;%PATH%
+
+install:
+  - cinst gow
+  - cinst php
+  - cinst composer
```

Comment:
@nicolas-grekas I suggest downloading the composer phar directly rather than using the windows installer for now, to have a non-interactive setup.